src/controllers/expenses.js

```diff
@@ -0,0 +1,89 @@
+'use strict';
+
+const expenseService = require('../services/expenses');
+const { getUserById } = require('../services/users');
+
+const createExpense = (request, response) => {
+  const params = request.body;
+
+  const foundUser = getUserById(params.userId);
+
+  if (!foundUser || !params) {
+    response.sendStatus(400);
+
+    return;
+  }
+
+  const newExpense = expenseService.createExpense(params);
+
+  response.statusCode = 201;
+  response.send(newExpense);
+};
+
+const getExpenses = (request, response) => {
+  const params = request.query;
+
+  const foundExpenses = expenseService
+    .getExpenses(params);
+
+  response.send(foundExpenses);
+};
+
+const getExpense = (request, response) => {
+  const { id } = request.params;
+  const expense = expenseService.getExpensesById(id);
```

Comment:
If the function return expense`s` variable should name expense`s`

const `expense` = expenseService.get`Expenses`ById(id);